README

```diff
@@ -103,8 +103,8 @@ the server settings as described in the "configure dirtsand" step.
 
 3) Set up the dirtsand database:
 
-   $ sudo -u postgres psql -d dirtsand < db/dbinit.sql
-   $ sudo -u postgres psql -d dirtsand < db/functions.sql
+   $ psql -d dirtsand < db/dbinit.sql
```

Comment:
These should still be done as the dirtsand user (`sudo -u dirtsand` instead of `sudo -u postgres`)